Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -765,4 +773,13 @@ extension ChatMessageListVC: AnnouncementAction {
     func didSelectAnnouncementAction(_ message: ChatMessage?) {
         debugPrint(message?.text)
```

Comment:
@parth-simformsolutions  , Could you please remove this method or remove debug print statement if it is not in use , thanks